File: Python3/1180 - Menor e Posição.py
CONTADOR = 0
N = int(input())
POSICAO = 0
X = list(map(int, input().split()))
MINIMO = X[0]
for i in X:
    CONTADOR += 1
    if MINIMO > i:
        MINIMO = i
        POSICAO = CONTADOR
print('Menor valor: {}'.format(MINIMO))
print('Posicao: {}'.format(POSICAO - 1))

File: C#/1180 - Menor e Posição.cs
using System; 

class URI {

    static void Main(string[] args) { 

        int menor = 1000, posicao = 0;
    int N = int.Parse(Console.ReadLine());
		string [] array = Console.ReadLine().Split();

    for (int i = 0; i < N; i++) {

			if (menor > int.Parse(array[i])) {
       	menor = int.Parse(array[i]);
    	  posicao = i;
      }		 

		}
    Console.WriteLine("Menor valor: {0}" , menor);
		Console.WriteLine("Posicao: {0}", posicao);

    }

}